src/cc_catalog_airflow/templates/workflow.py_template

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+from provider_api_scripts import cleveland_museum_of_art
+from util.operator_util import get_log_operator
+
+
+DAG_DEFAULT_ARGS = {
+    'owner': 'data-eng-admin',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 1, 15),
+    'email_on_retry': False,
+    'retries': 3,
+    'retry_delay': timedelta(days=1),
+}
```

Comment:
The [dag_factory][dag_factory_link] already use almost the same default args as here, from a [config][config_link] file. The workflow file can be slightly smaller and repeat less code if we use the factory, as the workflow script for the Smithsonian Institute and Flickr for example. Have you considered this?

[dag_factory_link]: https://github.com/WordPress/openverse-catalog/blob/template/src/cc_catalog_airflow/dags/util/dag_factory.py
[config_link]: https://github.com/WordPress/openverse-catalog/blob/template/src/cc_catalog_airflow/dags/util/config.py#L7:L14

Reply:
No, I haven't looked into the dag_factory, thank you for the suggestion, I will look into it!